apps/app/src/styles.css

```
@@ -40,6 +40,8 @@
 
 @custom-variant fixed (&:is(.layout-fixed *));
 
+@custom-variant logo-legacy (&:is(.logo-legacy *));
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)